### DISCUSSION TITLE
grad in model.named_parameters() are None

### DISCUSSION THREAD

**Discussion by asudeeaydin** [fangwei123456/SpikingFlow#234] [Q&A]
Hi, 

I'm trying to plot the gradients of the network, but when I looped over the layers, I realized that grad was None. This made me a bit worried about the training altogether.

Maybe this is expected since a surrogate gradient function is used? In that case, is there no way to plot the gradients?

**Comment by fangwei123456** [comment]
Can you show me the minimul codes to reproduce your problem?

**Reply by asudeeaydin** [reply]
When trying to condense the code I found the mistake, it was due to the loss function. 

Thanks for your quick response, really appreciate it!
